Doc/library/argparse.rst

```diff
@@ -354,6 +356,57 @@ including argument descriptions. However, multiple newlines are replaced with
 one. If you wish to preserve multiple blank lines, add spaces between the
 newlines.
 
+:class:`FlexiHelpFormatter` wraps description and help text like the default
```

Comment:
We should also make it more clear that consecutive non-blank lines at the same indentation without supported bullet markers are merged into one paragraph. Users should add blank lines if they want to preserve formatting.